### PR TITLE
ip04349

### PR DESCRIPTION
**Pull request by shrimp11**

### CODE REVIEW

File: target/linux/ramips/dts/mt7621_pangubox_ip04349.dts [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 / {
 	compatible = "pangubox,ip04349", "mediatek,mt7621-soc";
-	model = "PanguBox IP04349";
+	model = "PanguBox IP04316";
```
Comment (KyrinWrt) [inline target/linux/ramips/dts/mt7621_pangubox_ip04349.dts:10]: 你这里没改对，应该是IP043491

**Comment by KyrinWrt** [comment]
Hi!!
Thanks for contributing to OpenWRt!
Would you mind adding installation / recovery instructions to your commit message?
Thanks!

File: target/linux/ramips/dts/mt7621_pangubox_ip04349.dts (hunk 1 above)
Comment (KyrinWrt) [inline target/linux/ramips/dts/mt7621_pangubox_ip04349.dts:10]: 你这句model = "PanguBox IP04316";应该改为model = "PanguBox IP043491";

**Comment by shrimp11** [comment]
model = "PanguBox IP043491";

**Review by shrimp11 — COMMENTED** [review]
model = "PanguBox IP043491";

**Pull request merged**